src/utils/index.js

```diff
@@ -157,7 +156,7 @@ export const ACITriggerSettings = ({
         onReady(data, query);
       });
     },
-    component: MentionsItem,
+    component: 'MentionsItem',
```

Comment:
sure about this?

Reply:
yeah before we were saving the actual components in state and passing down as props, which doesn't work for functional components, so now doing a switch statement in the SuggestionsList component and rendering the correct component based on the type is receives via props, not the actual React component

Reply:
We have some users who are using their own settings - `triggerSettings` prop on AutoCompleteInput component. If we switch to strings, then custom settings won't work since [here](https://github.com/GetStream/stream-chat-react-native/pull/280/files#diff-ebdd5502ddc4752adf79bbf860158265R73) we only handle `MentionsItem` and `CommandsItem`. Can we do this some other way? 
(Breaking change is not an issue, but we need to be able to handle custom `triggerSettings`)

Reply:
Ok sounds good, I'll look into this a bit. As of now, the functional components don't like holding JSX in state like this and passing down to children, so either way we'll need to refactor some things

Reply:
Sadly, this change did break our implementation. Triggering a command (`/`, `@`) causes the app to crash after an upgrade to 1.2.0. I originally did the upgrade because `/giphy` images weren't showing up until you start typing in the textarea again.

Reply:
It appears our custom suggestion component doesn't work either. I see handling for custom components in the code but it still throws an exception.

Reply:
Hey Brandon, sorry to hear this. Are you guys working with class or function based components in your app? I noticed that upon the conversion to functions, passing a component around in this method began to cause problems, so I'm wondering if you're running into something similar. Let me know how you have your code set up, and I'll work on getting this patched ASAP.

Reply:
Brandon, I have a fix for this. Making a PR shortly and we'll get this into a release ASAP, thanks for noticing!

Reply:
@DanC5 I didn't see your response for some reason. Anyway, we're using a function based component. I upgraded `stream-chat-react-native` and `stream-chat` to the latest versions and it was still broken. Did your fix make it into a release?

Reply:
Hey Brandon, this fix has not yet been released, but it's in our master branch and will be deployed with the next round